README.md

```diff
@@ -2,60 +2,106 @@
-| [`@turnkey/iframe-stamper`](/packages/iframe-stamper)                             | [![npm](https://img.shields.io/npm/v/@turnkey/iframe-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/iframe-stamper)                             | Provide API key signatures over Turnkey requests within iframe contexts | [CHANGELOG](/packages/iframe-stamper/CHANGELOG.md)               | Turnkey primitives |
-| [`@turnkey/webauthn-stamper`](/packages/webauthn-stamper)                         | [![npm](https://img.shields.io/npm/v/@turnkey/webauthn-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/webauthn-stamper)                         | Provide Webauthn signatures over Turnkey requests                       | [CHANGELOG](/packages/webauthn-stamper/CHANGELOG.md)             | Turnkey primitives |
-| [`@turnkey/wallet-stamper`](/packages/wallet-stamper)                             | [![npm](https://img.shields.io/npm/v/@turnkey/wallet-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/wallet-stamper)                             | Provide wallet signatures over Turnkey requests                         | [CHANGELOG](/packages/wallet-stamper/CHANGELOG.md)               | Turnkey primitives |
+## Overview
+
+The Turnkey SDK includes functions and clients with methods that construct, authenticate and execute requests to the Turnkey API with support for different ecosystems and methods of authentication. It consists of the [Primary Turnkey Web SDK Packages](#primary-turnkey-web-sdk-packages) which expose the main functionality required to build Turnkey-powered applications in different web environments, the [Chain/Ecosystem-Specific Signing Packages](#chainecosystem-specific-signing-sdk-packages) which expose signers with support for specific ecosystems, built on top of our Web SDK, and the [Advanced Functionality SDK Packages](#advanced-functionality-sdk-packages) which exposes lower level functionality that is leveraged by our Primary Web SDK Packages for those with highly-specific implementations looking to use them.
+
+The below diagram helps visualize the packages in our SDK organized by the functionality they expose.
+
+<img src="./img/sdk-map.png" alt="homepage screenshot" width="1000px">
+
+## Primary Turnkey Web SDK Packages
+
+The following packages expose the main functionality required to build Turnkey-powered applications. Each package exposes functions, and/or client classes with methods that manage the process of authenticating requests to the Turnkey API in the contexts of a generic browser environment, a react or react native client environment, or a server environment.
```

Comment:
we do not have a react native client. We simply use the "normal" clients. Passkey stamping has to involve magic native APIs which is why we have a separate package for it, but HTTP calls into Turnkey work the same as on the web.

Reply:
This makes sense -- appreciate the color here! Had the same intuition because I noticed that it was just a stamper and that we just say to use the stamper along with the basic http client from @turnkey/http.


I ended up grouping it with the other larger libraries because it gets its own page in our docs after sdk-browser, sdk-server and sdk-react but definitely agree that it should go with the stampers and is more analogous with them